R/tm_a_pca.R

```diff
@@ -570,7 +586,7 @@ srv_a_pca <- function(id, data, reporter, filter_panel_api, dat, plot_height, pl
               ggthemes +
               themes
 
-            print(g)
+            print(plot)
```

Comment:
This needs to be evaluated after the "decoration", otherwise the code looks wrong:

![Uploading image.png…]()

Reply:
good catch